python/server/auto_generated/qgsserverapiutils.sip.in

```diff
@@ -35,6 +35,55 @@ Parses a comma separated ``bbox`` into a (possibily empty) :py:class:`QgsRectang
    Z values (i.e. a 6 elements bbox) are silently discarded
 %End
 
+    static QList< QgsVectorLayerServerProperties::WmsDimensionInfo > temporalDimensions( const QgsVectorLayer *layer );
+%Docstring
+Returns a list of temporal dimensions information for the given ``layer`` (either configured in wmsDimensions or the first date/datetime field)
+
+.. versionadded:: 3.12
+%End
+
+
+    struct TemporalDateInterval
+    {
+      QDate begin;
+      QDate end;
+    };
+
+    struct TemporalDateTimeInterval
```

Comment:
There's already a QgsDateTimeInterval class

Reply:
Sorry, I mean QgsDateRange and QgsDateTimeRange

Reply:
I saw that class before but it seemd overkill to me for this use case (just return a pair of values), but I'm ok to reconsider its use.